pom.xml

```diff
@@ -3514,7 +3514,7 @@
                         <configuration>
                             <rules>
                                 <requireMavenVersion>
-                                    <version>3.6.0</version>
+                                    <version>3.8.1</version>
```

Comment:
why not up to the latest (3.9.6) ?

Reply:
can try on that

Reply:
Anyway, it is likely just a minimal Maven version to support Jdk21, no special reason behind that. Can just work well with the latest 3.9.6 I guess.

Reply:
ok thanks for the explanation